math/geometry/distancebetweenpoints.go

```diff
@@ -0,0 +1,53 @@
+// distancebetweenpoints.go
+// Find distance between two points
+// Details:
+// Given coordinates of 2 points on a cartesian plane, find the distance between them rounded up to nearest integer.
+// Link: https://practice.geeksforgeeks.org/problems/distance-between-2-points3200/1?page=1&category[]=geometric
+
+// author(s) [Chetan Patil](https://github.com/Chetan07j)
+
+// Package geometry contains geometric algorithms
+package geometry
+
+import (
+	"errors"
+	"math"
+)
+
+// Point defines a point with x and y coordinates.
+type CartesianPoint []float64
```

Comment:
```suggestion
type Point []float64
```

Cartesian points are only two dimensional. This is `n` dimensional.

Reply:
type `Point` is already defined in package geometry in another file, so I could not name it the same here.

Reply:
Sure. Though I was thinking maybe the other one could be called `CartesianPoint` or `Coordinate`.

Reply:
Ok. Done made it as `EuclideanPoint`

Could you please review PR again?